.github/workflows/ci.yml

```diff
@@ -655,7 +674,10 @@ jobs:
           CURRENT_CHANGELOG=`tail --lines +2 changelog.md`
           echo -e "$CURRENT_CHANGELOG" >> changelog-new.md
           mv changelog-new.md changelog.md
-          pipenv run pre-commit run --files changelog.md || true
+          uv run \
```

Comment:
Mostly to test this, we might want to push a beta tag to ensure this portion runs before release

Reply:
Fine w me!